ecosystem/sep-0006.md

```diff
@@ -330,6 +344,8 @@ Name | Type | Description
 `on_change_callback` | string | (optional) A URL that the anchor should `POST` a JSON message to when the `status` property of the transaction created as a result of this request changes. The JSON message should be identical to the response format for the [/transaction](#single-historical-transaction) endpoint.
 `amount` | string | (optional) The amount of the asset the user would like to deposit with the anchor. This field may be necessary for the anchor to determine what KYC information is necessary to collect.
 `country_code` | string | (optional) The [ISO 3166-1 alpha-3](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) code of the user's current address. This field may be necessary for the anchor to determine what KYC information is necessary to collect.
+`destination_asset` | string | (optional) The off-chain asset the Anchor will deliver to the user's account. The value must match one of the `asset` values included in a [`SEP-38 GET /prices?sell_asset=stellar:<asset_code>:<asset_issuer>`](sep-0038.md#get-prices) response using [SEP-38 Asset Identification Format](sep-0038.md#asset-identification-format). If neither this field nor `quote_id` are set, it's assumed the anchor will be providing a bridge between equivalent assets, for instance receiving USDC in the Stellar network and depositing fiat USD to the user's bank account.
+`quote_id` | string | (optional) The `id` returned from a `SEP-38 POST /quote` response. If this attribute is specified, none of the attributes `asset_code`, `amount` or `destination_asset` should be included in the request. The Stellar transaction used to send the asset _must_ have a `started_at` timestamp that is earlier than the quote's `expires_at` attribute.
```

Comment:
> The Stellar transaction used to send the asset _must_ have a `started_at` timestamp that is earlier than the quote's `expires_at` attribute.

❗  There is no `started_at` attribute on the a Stellar transaction's record in Horizon, see the [docs](https://developers.stellar.org/api/resources/transactions/object/). I think you mean `created_at`. I saw you had a question about this on the SEP-31 PR, happy do discuss further.

Reply:
Good catch, thanks for spotting that! Here's the fix: https://github.com/stellar/stellar-protocol/pull/973/commits/c1a8a926f80e7c3a22efa7cc6c35ad8d350fcb23